base/deprecated.jl

```diff
@@ -240,4 +240,7 @@ end
 @deprecate cat_shape(dims, shape::Tuple{}, shapes::Tuple...) cat_shape(dims, shapes) false
 cat_shape(dims, shape::Tuple{}) = () # make sure `cat_shape(dims, ())` do not recursively calls itself
 
+@deprecate unsafe_indices(A) axes(A)
+@deprecate unsafe_length(r) length(r)
```

Comment:
These are currently not exported, so should be
```suggestion
@deprecate unsafe_indices(A) axes(A) false
@deprecate unsafe_length(r) length(r) false
```
I think.